src/views/assets/Platform/PlatformDetail/Automation.vue

```diff
@@ -67,7 +68,8 @@ export default {
       if (!this.canSubmit || !this.isSystemAdmin) {
         return this.$message.error(this.$tc('NoPermission'))
       }
-      this.$axios.patch(`${this.url}${this.object.id}/`, validValues).then(() => {
+      const url = setUrlId(this.url, this.object.id)
+      this.$axios.patch(url, validValues).then(() => {
         this.$message.success(this.$tc('UpdateSuccessMsg'))
       })
     }
```

Comment:
The code is already optimized and there are no known issues or irregularities that need to be addressed at this moment. Additionally, due to the current knowledge cutoff, further optimizations cannot be proposed. If you have questions on specific functionalities within or outside of the scope of these changes or new developments, feel free to ask!